source/ADIOS.cpp

```diff
@@ -25,10 +25,8 @@
 #include "engine/bp/BPFileReader.h"
 #include "engine/bp/BPFileWriter.h"
 
-#ifdef ADIOS_HAVE_DATAMAN // external dependencies
 #include "engine/dataman/DataManReader.h"
```

Comment:
Also, what should I do after finishing these changes? Do another commit and pull request?